### DISCUSSION TITLE
An error occured while communicating to the server.

### DISCUSSION THREAD

**Discussion by norwenz** [liedekef/events-made-easy#343] [Q&A] [answered]
After updating from 2.3.x to 2.4.14 the message can be seen in the backend: "An error occurred while communicating to the server."
I can't list the events and locations and therefore can't edit them anymore.

**Comment by liedekef** [comment, marked as answer]
Make sure you use PHP 8 or newer, older versions are not supported by EME (see also the changelog in readme.txt). 
If that is not the problem: open a new issue and provide the info as requested in the bug template
